src/uct/tcp/tcp_ep.c

```diff
@@ -1117,7 +1117,7 @@ static unsigned uct_tcp_ep_progress_magic_number_rx(uct_tcp_ep_t *ep)
     if (magic_number != UCT_TCP_MAGIC_NUMBER) {
         /* Silently close this connection and destroy its EP */
         ucs_debug("tcp_iface %p (%s): received wrong magic number (expected: "
-                  "%zu, received: %zu) for ep=%p (fd=%d) from %s", iface,
+                  "%zu, received: %"PRIu64 ") for ep=%p (fd=%d) from %s", iface,
```

Comment:
maybe also change "expected: %zu" to "expected: %lu"

Reply:
remove space after PRIu64

Reply:
Remove space and change to `%lu`
Does this mean the following?

`%"PRIu64 ")..."` -> `%"PRIu64") .."` If so I also modified addtional files.